src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinForgeVersion_FixUpdateCheck.java

```diff
@@ -0,0 +1,21 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
```

Comment:
Move this mixin to https://github.com/GTNewHorizons/Hodgepodge/blob/master/src/main/java/com/mitchej123/hodgepodge/mixins/early/forge